eventkit_cloud/jobs/models.py

```diff
@@ -136,7 +136,8 @@ class DataProvider(UIDMixin, TimeStampedModelMixin):
     slug = LowerCaseCharField(max_length=40, unique=True, default='')
     url = models.CharField(verbose_name="Service URL", max_length=1000, null=True, default='', blank=True,
                            help_text='The SERVICE_URL is used as the endpoint for WFS, OSM, and WCS services. It is '
-                                     'also used to check availability for all OGC services.')
+                                     'also used to check availability for all OGC services. If you are adding a TMS '
+                                     'service, please use the format http(s)://host/tiles/default')
```

Comment:
This is not `/tiles/default` standard and could change depending on the service provider.  The language should be explicit such as provide the full path with the z/y/x portion removed, or we could simply ask the user to put the full path to the files in a specific format which we could then parse later, `http(s)://host/tiles/default/{z}/{y}/{x}.png`